Sources/Orbit/Components/Collapse.swift

```diff
@@ -5,13 +5,13 @@ import SwiftUI
 /// - Note: [Orbit definition](https://orbit.kiwi/components/collapse/)
 public struct Collapse<Header: View, Content: View>: View {
 
-    struct _Collapse: View {
-        let header: Header
-        let headerVerticalPadding: CGFloat
-        let content: Content
-        @Binding var isExpanded: Bool
+    let header: Header
+    let headerVerticalPadding: CGFloat
+    let content: Content
+    let isExpanded: Binding<Bool>?
 
-        var body: some View {
+    public var body: some View {
+        BindingWrapper(isExpanded, defaultValue: false) { $isExpanded in
```

Comment:
Is it needed? We can probably now have a `var isExpanded: Binding<Bool>` and set it to dummy `get/set` in the init.

Reply:
I guess I can revert the changes here, but we'd be back to the inner type...

Reply:
I will check if what I suggest is technically possible

Reply:
I'd expect it is - just copy the relevant parts of `BindingWrapper` here.

To me the question is more whether there will be more components like `Collapse` here - where the binding is optional and state can be managed internally. Because if there will be more such views, it makes sense to me to extract this concept into a separate type - like `BindingWrapper`.

Reply:
The proposed `BindingSource`, `OptionalBinding` are fine. Or maybe something like `BindingResolver`, `BindingSwitcher` ?